backend/MoleculeOSSite/Program.cs

```diff
@@ -19,12 +57,19 @@
 }
 
 // Configure the HTTP request pipeline.
+
+var dbcontext = new MyDbContext();
+var seeder = new Seeder(dbcontext);
+seeder.SeedRole();

```

Comment:
```suggestion
seeder.SeedRoles();

```
Użyj liczby mnogiej ;)